### PR TITLE
CCR-1186: Tile for unsupported file types + watermark/PFP displays as a tile for incorrect file

### PR DESCRIPTION
**Pull request by riabokon-virtru**
…le type

### Proposed Changes
_Please use the Jira Key followed by the changes_

- JIRAKEY-1234
  - Some change
  - Another change

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [ ] I have verified that my changes have not introduced new lint errors
- [x] I have added a GitHub labels to the PR for either _patch_, `minor`, or **major**

### CODE REVIEW

File: lib/components/fileListItem/fileListItem.js [hunk 1]
```diff
@@ -17,60 +17,105 @@ const FileListItem = ({
...
+  persistentProtectionEnabled,
+  encryptedText,
+  watermarkText,
+  persistentProtectionText,
 }) => {
-  const tooltip = (
-    <>
-      <div className={styles.tooltipTitle}>
-        {warning && <WarningOutlined className={styles.warnIcon} />}
+  const tooltip = useMemo(() => {
```
Comment (ntrevino-virtru) [inline lib/components/fileListItem/fileListItem.js:27]: Can this be converted to a component?

Comment (riabokon-virtru) [reply lib/components/fileListItem/fileListItem.js]: Sure

**Review by ntrevino-virtru — APPROVED** [review]
:fire:

**Comment by riabokon-virtru** [comment]
@ntrevino-virtru can you merge this PR?

**Pull request merged**